### PR TITLE
Disallow empty cost model for create update proposal

### PR DESCRIPTION
**Pull request by newhoggy**
```
$ cardano-cli governance create-update-proposal --epoch 50 --max-block-execution-units '(20000000000,62000000)' --cost-model-file ~/Downloads/cost-model-secp-preprod-mainnet.json --protocol-major-version 8 --protocol-minor-version 0 --genesis-verification-key-file example/genesis-keys/genesis1.vkey --out-file update.proposal
Command failed: governance create-update-proposal  Error: The decoded cost model was empty at: /Users/jky/Downloads/cost-model-secp-preprod-mainnet.json
```

Resolves https://github.com/input-output-hk/cardano-node/issues/4867

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Run/Governance.hs [hunk 1]
```diff
@@ -52,8 +57,10 @@ renderShelleyGovernanceError err =
        <> " The number of staking keys: " <> textShow numVKeys
        <> " and the number of reward amounts: " <> textShow numRwdAmts
        <> " are not equivalent."
-    ShelleyGovernanceCmdCostModelsJsonDecodeErr err' fp ->
-      "Error decoding cost model: " <> Text.pack err' <> " at: " <> fp
+    ShelleyGovernanceCmdCostModelsJsonDecodeErr fp err' ->
```
Comment (newhoggy) [inline cardano-cli/src/Cardano/CLI/Shelley/Run/Governance.hs:60]: Swapping of the argument order is a bugfix.

File: update.proposal [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "type": "UpdateProposalShelley",
```
Comment (Jimbo4350) [inline update.proposal:2]: Left over file?

Comment (newhoggy) [reply update.proposal:2]: Removed.  Thanks!

**Review by Jimbo4350 — APPROVED** [review]

**Pull request merged**